pocs/camera/camera.py

```diff
@@ -1,13 +1,30 @@
 from .. import PanBase
-from ..focuser.focuser import AbstractFocuser
+
 from ..utils import error
 from ..utils import listify
 from ..utils import load_module
+from ..utils import images
+from ..utils import current_time
+
+from ..focuser.focuser import AbstractFocuser
+
+from astropy.io import fits
+from astropy.stats import sigma_clipped_stats
+from astropy.modeling import models, fitting
+
+import numpy as np
+
+import matplotlib
```

Comment:
Have moved the body of `autofocus()` and `_autofocus()` from `AbstractCamera` to `AbstractFocuser`. It could be argued that I should move the bulk of the autofocus tests from `test_camera.py` to `test_focuser.py` too, but then again you can't autofocus without both a `Camera` and a `Focuser`.